### ISSUE TITLE
Message: 

### ISSUE THREAD

**Issue by ZeusIssueReporter** [Ekultek/Zeus-Scanner#81]
Zeus version:
`1.0.39`

Error info:
```Traceback (most recent call last):
  File "/home/bush911/Desktop/Zeus-Scanner/var/google_search/search.py", line 248, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/home/bush911/Desktop/Zeus-Scanner/var/google_search/search.py", line 141, in get_urls
    retval = browser.current_url
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 563, in current_url
    return self.execute(Command.GET_CURRENT_URL)['value']
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 297, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
UnexpectedAlertPresentException: Alert Text: None
Message: 
````

Running details:
`Linux-4.12.0-parrot6-686-pae-i686-with-Parrot-3.8-JollyRoger`

Commands used:
`zeus.py --dork-list=/home/bush911/Desktop/d0rks.txt --blackwidow=URL http://notebase.net/29216-video_98.html , -A --sqlmap-args=SQLMAP-ARGS sqlmaap`

Log file info:
```2017-10-12 22:34:57,837;zeus-log;INFO;log file being saved to '/home/bush911/Desktop/Zeus-Scanner/log/zeus-log-44.log'...
2017-10-12 22:34:57,837;zeus-log;WARNING;AOL will take a little longer due to pop-ups...
2017-10-12 22:34:57,837;zeus-log;INFO;specified to use non-default search engine...
2017-10-12 22:34:57,840;zeus-log;INFO;starting dork scan with query '# See LICENSE for license details.'...
2017-10-12 22:34:57,841;zeus-log;INFO;attempting to gather query URL...
2017-10-12 22:34:57,986;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-12 22:34:57,986;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-12 22:35:13,219;zeus-log;INFO;browser will open shortly...
2017-10-12 22:36:09,067;zeus-log;INFO;searching 'http://aol.com' using query '# See LICENSE for license details.'...
2017-10-12 22:36:13,061;zeus-log;INFO;closing the browser and continuing process..
2017-10-12 22:36:13,156;zeus-log;INFO;URL successfully gathered, searching for GET parameters...
2017-10-12 22:36:13,156;zeus-log;INFO;no proxy configuration detected...
2017-10-12 22:36:19,751;zeus-log;INFO;adjusting user-agent header to default user agent 'Zeus-Scanner(v1.0.39)::Python->v2.7'...
2017-10-12 22:36:19,897;zeus-log;INFO;found a total of 185 URL's with a GET parameter...
2017-10-12 22:36:19,898;zeus-log;INFO;successfully wrote found items to '/home/bush911/Desktop/Zeus-Scanner/log/url-log/url-log-1.log'...
2017-10-12 22:36:19,899;zeus-log;INFO;starting dork scan with query '# This file is part of v3n0m'...
2017-10-12 22:36:19,899;zeus-log;INFO;attempting to gather query URL...
2017-10-12 22:36:20,041;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-12 22:36:20,041;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-12 22:36:34,130;zeus-log;INFO;browser will open shortly...
2017-10-12 22:37:17,737;zeus-log;INFO;searching 'http://aol.com' using query '# This file is part of v3n0m'...
2017-10-12 22:37:21,756;zeus-log;INFO;closing the browser and continuing process..
2017-10-12 22:37:21,872;zeus-log;INFO;URL successfully gathered, searching for GET parameters...
2017-10-12 22:37:21,873;zeus-log;INFO;no proxy configuration detected...
2017-10-12 22:37:23,516;zeus-log;INFO;adjusting user-agent header to default user agent 'Zeus-Scanner(v1.0.39)::Python->v2.7'...
2017-10-12 22:37:23,523;zeus-log;INFO;found a total of 5 URL's with a GET parameter...
2017-10-12 22:37:23,523;zeus-log;INFO;successfully wrote found items to '/home/bush911/Desktop/Zeus-Scanner/log/url-log/url-log-2.log'...
2017-10-12 22:37:23,523;zeus-log;INFO;starting dork scan with query '********.asp?cid='...
2017-10-12 22:37:23,524;zeus-log;INFO;attempting to gather query URL...
2017-10-12 22:37:23,665;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-12 22:37:23,666;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-12 22:37:33,543;zeus-log;INFO;browser will open shortly...
2017-10-12 22:38:19,963;zeus-log;INFO;searching 'http://aol.com' using query '********.asp?cid='...
2017-10-12 22:38:23,855;zeus-log;INFO;closing the browser and continuing process..
2017-10-12 22:38:23,945;zeus-log;INFO;URL successfully gathered, searching for GET parameters...
2017-10-12 22:38:23,946;zeus-log;INFO;no proxy configuration detected...
2017-10-12 22:38:29,241;zeus-log;INFO;adjusting user-agent header to default user agent 'Zeus-Scanner(v1.0.39)::Python->v2.7'...
2017-10-12 22:38:29,349;zeus-log;INFO;found a total of 185 URL's with a GET parameter...
2017-10-12 22:38:29,349;zeus-log;INFO;successfully wrote found items to '/home/bush911/Desktop/Zeus-Scanner/log/url-log/url-log-3.log'...
2017-10-12 22:38:29,350;zeus-log;INFO;starting dork scan with query '********.php?cid='...
2017-10-12 22:38:29,350;zeus-log;INFO;attempting to gather query URL...
2017-10-12 22:38:29,495;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-12 22:38:29,496;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-12 22:38:39,306;zeus-log;INFO;browser will open shortly...
2017-10-12 22:39:24,652;zeus-log;INFO;searching 'http://aol.com' using query '********.php?cid='...
2017-10-12 22:39:28,484;zeus-log;INFO;closing the browser and continuing process..
2017-10-12 22:39:28,571;zeus-log;INFO;URL successfully gathered, searching for GET parameters...
2017-10-12 22:39:28,571;zeus-log;INFO;no proxy configuration detected...
2017-10-12 22:39:34,137;zeus-log;INFO;adjusting user-agent header to default user agent 'Zeus-Scanner(v1.0.39)::Python->v2.7'...
2017-10-12 22:39:34,247;zeus-log;INFO;found a total of 185 URL's with a GET parameter...
2017-10-12 22:39:34,248;zeus-log;INFO;successfully wrote found items to '/home/bush911/Desktop/Zeus-Scanner/log/url-log/url-log-4.log'...
2017-10-12 22:39:34,249;zeus-log;INFO;starting dork scan with query '********.php?id='...
2017-10-12 22:39:34,249;zeus-log;INFO;attempting to gather query URL...
2017-10-12 22:39:34,389;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-12 22:39:34,390;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-12 22:39:44,544;zeus-log;INFO;browser will open shortly...
2017-10-12 22:40:33,508;zeus-log;INFO;searching 'http://aol.com' using query '********.php?id='...
2017-10-12 22:40:37,375;zeus-log;INFO;closing the browser and continuing process..
2017-10-12 22:40:37,465;zeus-log;INFO;URL successfully gathered, searching for GET parameters...
2017-10-12 22:40:37,465;zeus-log;INFO;no proxy configuration detected...
2017-10-12 22:40:42,790;zeus-log;INFO;adjusting user-agent header to default user agent 'Zeus-Scanner(v1.0.39)::Python->v2.7'...
2017-10-12 22:40:42,901;zeus-log;INFO;found a total of 183 URL's with a GET parameter...
2017-10-12 22:40:42,902;zeus-log;INFO;successfully wrote found items to '/home/bush911/Desktop/Zeus-Scanner/log/url-log/url-log-5.log'...
2017-10-12 22:40:42,902;zeus-log;INFO;starting dork scan with query '********.php?pid='...
2017-10-12 22:40:42,903;zeus-log;INFO;attempting to gather query URL...
2017-10-12 22:40:43,046;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-12 22:40:43,046;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-12 22:40:55,569;zeus-log;INFO;browser will open shortly...
2017-10-12 22:41:30,288;zeus-log;INFO;searching 'http://aol.com' using query '********.php?pid='...
2017-10-12 22:41:34,174;zeus-log;INFO;closing the browser and continuing process..
2017-10-12 22:41:34,255;zeus-log;INFO;URL successfully gathered, searching for GET parameters...
2017-10-12 22:41:34,255;zeus-log;INFO;no proxy configuration detected...
2017-10-12 22:41:35,575;zeus-log;INFO;adjusting user-agent header to default user agent 'Zeus-Scanner(v1.0.39)::Python->v2.7'...
2017-10-12 22:41:35,583;zeus-log;INFO;found a total of 3 URL's with a GET parameter...
2017-10-12 22:41:35,583;zeus-log;INFO;successfully wrote found items to '/home/bush911/Desktop/Zeus-Scanner/log/url-log/url-log-6.log'...
2017-10-12 22:41:35,584;zeus-log;INFO;starting dork scan with query '********lang_englishlang_main_album.php?phpbb_root_path='...
2017-10-12 22:41:35,584;zeus-log;INFO;attempting to gather query URL...
2017-10-12 22:41:35,721;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-12 22:41:35,722;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-12 22:41:46,488;zeus-log;INFO;browser will open shortly...
2017-10-12 22:42:15,897;zeus-log;INFO;searching 'http://aol.com' using query '********lang_englishlang_main_album.php?phpbb_root_path='...
2017-10-12 22:42:20,207;zeus-log;ERROR;[7;31;31msearch.pyc failed to gather the URL from search engine, caught exception 'Alert Text: None
Message:' exception has been logged to current log file...
Traceback (most recent call last):
  File "/home/bush911/Desktop/Zeus-Scanner/var/google_search/search.py", line 248, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/home/bush911/Desktop/Zeus-Scanner/var/google_search/search.py", line 141, in get_urls
    retval = browser.current_url
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 563, in current_url
    return self.execute(Command.GET_CURRENT_URL)['value']
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 297, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
UnexpectedAlertPresentException: Alert Text: None
Message: 


2017-10-12 22:44:01,949;zeus-log;INFO;[32mZeus got an unexpected error and will automatically create an issue for this error, please wait...[0m
2017-10-12 22:44:01,950;zeus-log;INFO;[32mgetting authorization...[0m
2017-10-12 22:44:02,010;zeus-log;INFO;[32mextracting traceback from log file...[0m
```

**Comment by Ekultek** [comment]
Basically there was a popup and it wasn’t handled. I’m at the store so I’ll take care of this later good reference https://stackoverflow.com/questions/28397370/python-selenium-unexpectedalertpresentexception

**Comment by Ekultek** [comment]
Patched via 307751e, alright lets get into your arguments now;

`zeus.py --dork-list=/home/bush911/Desktop/d0rks.txt --blackwidow=URL http://notebase.net/29216-video_98.html , -A --sqlmap-args=SQLMAP-ARGS sqlmaap`

 - You cannot run two mandatory flags at one time, idk how it's expected to crawl one URL while it parse multiple pages of HTML for URL links, that's a lot to be asking of anyone
 - You have a random comma in your ARGV (argument variables) `html , -A` this will cause it to fail because that shouldn't be there, it's probably a typo but it's still not suppose to be there
 - `--sqlmap-args` is passed without passing a call to sqlmap's API `-s` and on top of that, idk what the hell `SQLMAP-ARGS` is, you're suppose to pass the arguments, if you don't know how to pass them you can see [here](https://github.com/Ekultek/Zeus-Scanner/wiki/Passing-flags-to-sqlmap) or run `--show-sqlmap` for a list of arguments.
 - You have an argument that says `sqlmaap` once again probably a typo, but it still shouldn't be there. To run sqlmap you need to do something like this:

```
baal@baal-Aspire-5733Z:~/bin/python/zeus-scanner$ sudo python zeus.py -f log/url-log/url-log-10.log --verbose -s --sqlmap-args="level 3, risk 3, randomAgent true, dropSetCookie true"
[18:04:18 DEBUG] checking if the application has been run before...
[18:04:18 DEBUG] verifying operating system...
[18:04:18 DEBUG] already ran, skipping...

    __          __________                             __   
   / /          \____    /____  __ __  ______          \ \  
  / /    ______   /     // __ \|  |  \/  ___/  ______   \ \ 
  \ \   /_____/  /     /\  ___/|  |  /\___ \  /_____/   / / 
   \_\          /_______ \___  >____//____  >          /_/  
                       \/   \/           \/  v1.0.42.aeb1(revision)
	https://github.com/ekultek/zeus-scanner.git
		Advanced Dork Searching...


[*] starting up at 18:04:18..


[18:04:18 DEBUG] running with options '{'runInVerbose': True, 'sqlmapArguments': 'level 3, risk 3, randomAgent true, dropSetCookie true', 'fileToEnumerate': 'log/url-log/url-log-10.log', 'runSqliScan': True}'...
[18:04:18 INFO] log file being saved to '/home/baal/bin/python/zeus-scanner/log/zeus-log-64.log'...
[18:04:18 DEBUG] using default search engine (Google)...
[18:04:18 PROMPT] would you like to process found URL: 'http://rover.ebay.com/rover/1/711-53200-19255-0/1?icep_ff3=9&pub=5575201205&toolid=10001&campid=5337930451&customid=&icep_sellerId=&icep_ex_kw=&icep_sortBy=12&icep_catId=&icep_minPrice=&icep_maxPrice=&ipn=psmain&icep_vectorid=229466&kwid=902099&mtid=824&kw=lg&icep_uq='[y/N]: y
[18:04:19 INFO] creating arguments for sqlmap...
[18:04:20 INFO] initializing new sqlmap scan with given URL 'http://rover.ebay.com/rover/1/711-53200-19255-0/1?icep_ff3=9&pub=5575201205&toolid=10001&campid=5337930451&customid=&icep_sellerId=&icep_ex_kw=&icep_sortBy=12&icep_catId=&icep_minPrice=&icep_maxPrice=&ipn=psmain&icep_vectorid=229466&kwid=902099&mtid=824&kw=lg&icep_uq='...
[18:04:20 DEBUG] scan initialized...
[18:04:20 INFO] gathering sqlmap API scan ID...
[18:04:20 DEBUG] current sqlmap scan ID: '805c5ce387a0e645'...
[18:04:20 INFO] starting sqlmap scan on url: 'http://rover.ebay.com/rover/1/711-53200-19255-0/1?icep_ff3=9&pub=5575201205&toolid=10001&campid=5337930451&customid=&icep_sellerId=&icep_ex_kw=&icep_sortBy=12&icep_catId=&icep_minPrice=&icep_maxPrice=&ipn=psmain&icep_vectorid=229466&kwid=902099&mtid=824&kw=lg&icep_uq='...
[18:04:20 DEBUG] using arguments: '{'dropSetCookie': 'true', 'randomAgent': 'true', 'risk': '3', 'level': '3'}'...
[18:04:20 INFO] adding arguments to sqlmap API...
[18:04:20 WARNING] please keep in mind that this is the API, output will not be saved to log file, it may take a little longer to finish processing, launching sqlmap...
------------------------------
sqlmap> [18:04:21 INFO] fetched random HTTP User-Agent header from file '/home/baal/bin/python/sqlmap/txt/user-agents.txt': 'Opera/9.80 (Windows NT 6.0; U; pl) Presto/2.7.62 Version/11.01'
sqlmap> [18:04:21 WARNING] provided value for parameter 'customid' is empty. Please, always use only valid parameter values so sqlmap could be able to run properly
sqlmap> [18:04:21 WARNING] provided value for parameter 'icep_sellerId' is empty. Please, always use only valid parameter values so sqlmap could be able to run properly
sqlmap> [18:04:21 WARNING] provided value for parameter 'icep_ex_kw' is empty. Please, always use only valid parameter values so sqlmap could be able to run properly
sqlmap> [18:04:21 WARNING] provided value for parameter 'icep_catId' is empty. Please, always use only valid parameter values so sqlmap could be able to run properly
sqlmap> [18:04:21 WARNING] provided value for parameter 'icep_minPrice' is empty. Please, always use only valid parameter values so sqlmap could be able to run properly
sqlmap> [18:04:21 WARNING] provided value for parameter 'icep_maxPrice' is empty. Please, always use only valid parameter values so sqlmap could be able to run properly
sqlmap> [18:04:21 WARNING] provided value for parameter 'icep_uq' is empty. Please, always use only valid parameter values so sqlmap could be able to run properly
sqlmap> [18:04:22 INFO] testing connection to the target URL
^C[18:04:23 ERROR] user aborted process...


[*] shutting down at 18:04:23..


baal@baal-Aspire-5733Z:~/bin/python/zeus-scanner$ 
```

Update to the newest version and try again
